R/bundle.R

```diff
@@ -316,6 +316,21 @@ writeBundle <- function(bundleDir, bundlePath, verbose = FALSE) {
   utils::tar(bundlePath, files = NULL, compression = "gzip", tar = tarImplementation)
 }
 
+
+getTarImplementation <- function() {
+  # Check the RSCONNECT_TAR environment variable first. If that is unset, check
+  # the rsconnect.tar option. If neither are set, use "internal"
+  tarImplementation <- Sys.getenv("RSCONNECT_TAR", unset = NA)
```

Comment:
The `rsconnect.ca.bundle` / `RSCONNECT_CA_BUNDLE` are evaluated in the opposite order - the option is used first.

Given that `rsconnect.tar` is new with this work, why not follow the same evaluation order?

Option takes precedence over the environment variable.

Reply:
Yeah, I'll make this change. I was unsure whether it'd be better to be consistent or to let the older option take precedence, but thinking about it now, I completely agree with you.